examples/gh-runner-gke-dind/outputs.tf

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "kubernetes_endpoint" {
+  description = "The cluster endpoint"
+  sensitive   = true
+  value       = module.runner-gke.kubernetes_endpoint
+}
+
+output "client_token" {
```

Comment:
Why do we need to output this?

Reply:
I use these in some of the inspec tests here. https://github.com/terraform-google-modules/terraform-google-github-actions-runners/blob/53beb1d26ba568732745a64f0d2b6c2158e4c8d7/test/integration/gh-runner-gke/controls/kubectl.rb#L19